charts/keycloakx/templates/poddisruptionbudget.yaml

```diff
@@ -1,5 +1,6 @@
 {{- if .Values.podDisruptionBudget -}}
-apiVersion: policy/v1beta1
+{{- if or (and .Values.autoscaling.enabled (gt (.Values.autoscaling.minReplicas | int) 1)) (and (not .Values.autoscaling.enabled) (gt (.Values.replicas | int) 1)) }}
```

Comment:
@bovy89 i'm not sure, if i like this restriction - at least without any part in the documentation describing this limitation.
Additionally, as the default is "no" PDB, i'd say it doesn't need to be double-checked here.

Reply:
That could be an additional safety measure because PDB with replica 1 is usually a wrong configuration. That is not strictly necessary...I can remove it if you want

Reply:
I'd say, let's drop this step, as it introduces new layers of dependecies to values etc. The updated API-Version is indeed a good thing to fix! :)